auth-api/migrations/versions/b80bbb12beda_add_entity_corp_sub_type.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
+
+Revision ID: b80bbb12beda
+Revises: e7c262b6f986
+Create Date: 2022-11-10 16:11:36.578577
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import Boolean, String
+from sqlalchemy.sql import column, table
+
+
+# revision identifiers, used by Alembic.
+revision = 'b80bbb12beda'
+down_revision = 'e7c262b6f986'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('entities', sa.Column('corp_sub_type_code', sa.String(length=15), nullable=True))
+    op.create_foreign_key(None, 'entities', 'corp_types', ['corp_sub_type_code'], ['code'])
+    corp_type_table = table('corp_types',
+                            column('code', String),
+                            column('description', String),
+                            column('default', Boolean)
+                            )
+    op.bulk_insert(
+        corp_type_table,
+        [
+            {'code': 'CC', 'description': 'BC Community Contribution Company', 'default': False},
+            {'code': 'BEN', 'description': 'BC Benefit Company', 'default': False}
+        ]
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(None, 'entities', type_='foreignkey')
```

Comment:
This doesn't reference to the constraint name? 

Couldn't it potentially drop these?
![image](https://user-images.githubusercontent.com/3484109/201707045-073d66b2-fb16-4549-bef4-e9709a813c96.png)